source/load-balancers/load-balancers.rst

```diff
@@ -367,22 +361,26 @@ order to not mix intentions.
 Why does this specification not address load balancer restarts or maintenance?
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-It was decided that the core problem that was needed to be solved was to introduce durable/resumable
-cursors to some services. The related ticket is `DRIVERS-1549 <https://jira.mongodb.org/browse/DRIVERS-1549>`__.
+The Layer 4 load balancers that would be in use for this feature lack the
+ability that a layer 7 load balancer could potentially have to be able to
+understand the MongoDB wire protocol and respond to monitoring requests.
+
+Why was connection pinning for cursors added to this spec?
```

Comment:
I'm not following how resumable cursors are related to connection pinning behavior. The former would make cursors more robust while the latter is needed to ensure cursor-related commands are all sent to the same backing server. Also, durable/resumable cursors would be useful in all contexts, not just LB mode. Can we remove this paragraph for simplicity?

Reply:
I've removed this section now. I wanted to keep it around for context on discussions we had but I think since it's still in the google doc it was ok to remove here.